apps/web/components/ReviewsAccordion/ReviewsAccordion.vue

```diff
@@ -187,13 +183,17 @@ async function fetchReviews() {
   ]);
 }
 
-const deleteReview = () => {
-  fetchReviews();
-  emits('on-list-change');
-};
-const maxVisiblePages = computed(() => (viewport.isGreaterOrEquals('lg') ? 10 : 1));
+const deleteReview = () => fetchReviews();
 
-onMounted(() => fetchReviews());
+onNuxtReady(() => fetchReviews());
```

Comment:
nuxtReady is fetching Reviews on client, should't we fetch reviews on server for SEO lighthouse score ?